docs/operation/volume_management/storageclass_creation.rst

```diff
@@ -0,0 +1,51 @@
+StorageClass Creation
+=====================
+
+**MetalK8s** uses **StorageClass** to configure how **Volumes** are formatted
+and mounted.
+This section hightlights how to create a Storageclass using the **CLI**.
+
+#. Create a StorageClass manifest.
+
+   You can define a new **StorageClass** using the following template:
+
+   .. code-block:: yaml
+
+       apiVersion: storage.k8s.io/v1
+       kind: StorageClass
+       metadata:
+         name: <storageclass_name>
+       provisioner: kubernetes.io/no-provisioner
+       reclaimPolicy: Retain
+       volumeBindingMode: WaitForFirstConsumer
+       mountOptions:
+         - rw
+       parameters:
+         fsType: <filesystem_type>
+         mkfsOptions: '["-m", "0"]'
+
+   Set the following fields:
+
+      - **mountOptions**: specifies how the volume should be mounted. For
+        example **rw** (read/write), **ro** (read-only).
+      - **fsType**: specifies the filesystem to use on the volume. For example
+        **XFS**, **ext4**.
```

Comment:
Do we already have documentation on the currently supported FS? If not, this is a good candidate for it.

Reply:
should be `xfs` BTW, we are case-sensitive.

And what do you mean about "documentation on the supported FS"? What could/should we say more than "we support Foo and Bar"?

Reply:
We only support this two for now

Reply:
The phrase now reads

```
- **fsType**: specifies the filesystem to use on the volume.
        **XFS** and **ext4** are the only currently supported file system types.```

Reply:
Please use `xfs` instead of `XFS`.

Even if the correct typo is indeed `XFS`, the value you can put in the YAML is lowercase only.